### PR TITLE
Feature/sqone 270/site footer

### PR DESCRIPTION
**Pull request by dpellenwood**
## What does this do/fix?

Builds out an MVP site footer. 🦶 

## QA

Links to relevant issues
- Footer: [SQONE-270](https://moderntribe.atlassian.net/browse/SQONE-270)
- Footer Nav: [SQONE-521](https://moderntribe.atlassian.net/browse/SQONE-521)

Screenshots:
- [Site Footer](http://p.tri.be/gG1rDt)
- [Site Footer Settings](http://p.tri.be/Z0Uqhl)

Demo:
- [Homepage w/ Footer](https://sq1-pr1013.moderntribe.qa/)
- [Footer Settings](https://sq1-pr1013.moderntribe.qa/wp-admin/themes.php?page=theme-options)

## Tests

Does this have tests?

- [ ] Yes
- [x] No, this doesn't need tests because it's boilerplate template development
- [ ] No, I need help figuring out how to write the tests.

### CODE REVIEW

**Review by Vinsanity — APPROVED** [review]
Looks good from my perspective Dave. GHA is yelling at you however. 😂

**Review by ChrisMKindred — APPROVED** [review]
👍

**Review by lindseyolson — APPROVED** [review]

**Review by Vinsanity — APPROVED** [review]
Nice work here Davee! Is this something Design is already aware of or we this informed by something they already provided?

**Review by defunctl — CHANGES_REQUESTED** [review]

**Comment by dpellenwood** [comment]
> Nice work here Davee! Is this something Design is already aware of or we this informed by something they already provided?

Yep. It's already in the [SquareOne Figma](https://www.figma.com/file/Wj2g7HVyBQWYIuxgNKMCxf/SQ1-Main-Patterns?node-id=5417%3A6431) file. ;)

**Review by ChrisMKindred — APPROVED** [review]

File: wp-content/themes/core/components/site_footer/Site_Footer_Controller.php [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+		return strpos( get_post_mime_type( $image_id ), 'image/svg' )
+			? $this->get_logo_image_svg_src( $image_id )
+			: $this->get_logo_image_default_src( $image_id );
+	}
+
+
+	private function get_logo_image_svg_src( int $image_id ): string {
+		$image_path = get_attached_file( $image_id );
+
+		return file_exists( $image_path ) ? file_get_contents( $image_path ) : '';
```
[the remark this answers is not in the document]
Comment (defunctl) [reply wp-content/themes/core/components/site_footer/Site_Footer_Controller.php]: @dpellenwood what we should do is that just after this is image is uploaded, capture its source then (one time) and save it to the options table. Any time you need the source, fetch it using get_option() instead and also probably a good idea to run it through the SVG sanitization library we have in tribe libs once more before displaying it

[the remark this answers is not in the document]
Comment (dpellenwood) [reply wp-content/themes/core/components/site_footer/Site_Footer_Controller.php]: Thanks Justin. Chatted w/ Chris about this and decided to pull it from the Footer. We'll just render the SVG as an image for default SquareOne. If a project needs something more complex, we can handle it then.

**Pull request merged**